message_ix_workshop.rst

```diff
@@ -5,50 +5,100 @@ The aim of the workshop is to help new users of the MESSAGEix
 modelling framework to get started with their modeling work.
 The main features of the “framework” are introduced, and the
 use cases of some features are shown. The user can learn how
-to build an energy model and how to represent some policy constraints
-in their energy scenarios.
+to build an energy model and how to represent some policy
+constraints in their energy scenarios.
 For information about the model, its structure, mathematical
-formulation and much more, please see the documentation
-at: https://docs.messageix.org.
-The different lectures contain the workshop slides, videos as
-well as tutorials for hands-on examples.
+formulation and much more, please see the `documentation <https://docs.messageix.org>`_.
 
-Introduction to MESSAGEix Modelling Framework
-"""""""""""""""""""""""""""""""""""""""""""""
-Introduction to the MESSAGEix modeling framework and its components.
+Under the following link you find the workshop slides as well
+as the recordings of the latest workshop:
 
+`MESSAGEix-Workshop material <http://pure.iiasa.ac.at/id/eprint/17318/>`_
 
-Getting started with the framework.
+Learning objectives
+"""""""""""""""""""
 
+The learning objectives of the workshop are the following:
 
-Building a simple MESSAGEix model
-"""""""""""""""""""""""""""""""""
-Introduction to linear optimization and to energy systems.
+* Understanding of energy systems.
```

Comment:
Understanding can be both a verb and a noun, so I would start this list with an phrase that can only be a verb to avoid a "step change" in the rhythm of this list. 
```suggestion
* Gaining a good understanding of energy systems modelling.
```